spec/card_spec.rb

```diff
@@ -220,6 +220,11 @@ module Trello
       it "gets its pos" do
         expect(card.pos).to_not be_nil
       end
+
+      it "gets its creation date" do
```

Comment:
can you please assert against a static value? I could not tell what value it would produce.